src/components/Common/Pagination.tsx

```
@@ -85,20 +85,46 @@ const Pagination = ({
         <NavButton
           id="prev-page"
           tooltip="Previous"
           onClick={() => goToPage(currentPage - 1)}
           disabled={currentPage - 1 <= 0}
           children={<CareIcon icon="l-angle-left" className="text-lg" />}
         />
+        <NavButton
+          id={`page-${currentPage}`}
+          key={currentPage}
+          onClick={() => goToPage(currentPage)}
+          selected={currentPage === currentPage}
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)